changes.md

```diff
@@ -0,0 +1 @@
+//
```

Comment:
This is your second review comment.

Make some changes to the file or remove it (`git rm changes.md`).

Reply:
I`ve used "git rm changes.md" and removed this file from local repo. But it stay in global. How i could change "changes.md"?

Reply:
1. Commit your change. `git commit -m "remove changes.md file"`
1. Push your local changes to the GitHub. Do `git push` or use your IDE for that. 
1. You should see a new commit here once it's pushed and the page is refreshed. 

By the way, do you rely on command line commands to do everything, or do you use your IDE features to do it? Just so I know for the future.

Reply:
Ok, it helped to me. I use the command line.